src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -820,11 +820,18 @@ internal override SyntaxNode AsInterfaceMember(SyntaxNode m)
             DeclarationModifiers modifiers,
             IEnumerable<SyntaxNode> members)
         {
+            return EnumDeclaration(name, null, accessibility, modifiers, members);
+        }
+
+        internal override SyntaxNode EnumDeclaration(string name, SyntaxNode underlyingType, Accessibility accessibility = Accessibility.NotApplicable, DeclarationModifiers modifiers = default, IEnumerable<SyntaxNode> members = null)
+        {
+            List<BaseTypeSyntax> underlyingTypeList = underlyingType != null ? new() { SyntaxFactory.SimpleBaseType((TypeSyntax)underlyingType) } : null;
```

Comment:
there can only be at most one base type, so i don't see the need for the list.

Reply:
Ah, I thought I needed the list to create the separated list with the syntax factory, but you mentioned how to do it in another comment. Thanks :)